src/op_analytics/datapipeline/models/templates/erc20_contracts/superchainerc20.sql.j2

```diff
@@ -0,0 +1,36 @@
+WITH create2 AS (
+  SELECT
+    *
+    , 'SuperchainERC20' AS token_type
+  FROM {{ contract_creation_traces }}
+  WHERE
+    -- crosschainBurn(address,uint256): 0x2b8c49e3
+    -- crosschainMint(address,uint256): 0x18bf5077
+    (output LIKE '%2b8c49e3%' AND output LIKE '%18bf5077%')
```

Comment:
Should we rename all of this to ERC____ (whatever the crosschainMint/Burn standard is?)

Since we actually need to call these functions on the contract to check the address with mint/burn rights to confirm if it's SuperchainERC20

Reply:
yea can do that. 
note another contract call necessary here to call and verify the actual mint and burn address
it's erc7802: https://docs.optimism.io/stack/interop/superchain-erc20

also lemme follow up with the product team next week if it's even possible to know the contract address at all